### DISCUSSION TITLE
Improve Docs and examples

### DISCUSSION THREAD

**Discussion by AlexNetman** [artilleryio/artillery#1832] [Artillery]
1) I have found Artillery when searching some way to run Performance test with AWS Serverless.
Only after a white I have understood that this feature is only for paid version. Which is NOT available at the moment.
Say it clearly on Artillery homepage and github.  "Free - only local"

2) Local running option is NOT clear. No example config where I can adjust Ramping, and Leveled load.
Add it to the Github homepage. Add example.yml with each line self-explained config with comments.
Add some comments into code even from the code I can't understand how to configure leveled phase. How?

**Comment by hassy** [comment]
hi @AlexNetman, thanks for the feedback.

Serverless load testing on AWS Lambda is available out-of-the-box in Artillery. You can install Artillery and run a test on AWS Lambda in less than 5 minutes if you have your AWS credentials set up. The full guide is here: https://www.artillery.io/docs/guides/guides/distributed-load-tests-on-aws-lambda

We have a guide that takes you through building up a test script from scratch:

https://www.artillery.io/docs/guides/getting-started/writing-your-first-test

As well as a reference guide for Artillery scripts:

https://www.artillery.io/docs/guides/guides/test-script-reference

There's also a bunch of end-to-end examples linked from the README: https://github.com/artilleryio/artillery/tree/main/examples

Would love more specific feedback on what could be explained better.

**Comment by hassy** [comment]
moving to discussions
